### PR TITLE
Improved README.

### PR DESCRIPTION
**Pull request by SirVer**
This change is [Reviewable](https://reviewable.io/reviews/widelands/widelands_metaserver/11)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,6 +1,23 @@
...
+4. `cd src/github.com/widelands/widelands_metaserver/wlms`
+5. `make`
+
+# Deploying
+
+1. `make cross`
+2. scp `wlms` over to the server and replace `/usr/local/bin/wlms`.
+3. `sudo restart wlmetaserver`
+4. Check in `/var/log/upstart/wlmetaserver.log` that the restart was
+   successfull.
```
Comment (gunchleoc) [inline README.md]: successfull -> successful

**Comment by gunchleoc** [comment]
Just 1 tiny English language nit.

**Comment by SirVer** [comment]
> Just 1 tiny English language nit.

I think you did not finish the review. The comments only show up once you send them in bulk.

**Review by gunchleoc — APPROVED** [review]

**Comment by gunchleoc** [comment]
You're right, I didn't see the big shiny green button that they wanted me to press.

**Pull request merged**